### PR TITLE
fix: Sidebar context menu behaviour (WEBAPP-4312)

### PR DESCRIPTION
**Pull request by Yserz**

### CODE REVIEW

File: electron/renderer/src/actions/index.js [hunk 1]
```diff
@@ -45,7 +45,21 @@ export const deleteAccount = id => {
   };
 };
 
-export const abortAccountCreation = (id) => {
+export const toggleAccountContextVisibility = (x, y) => {
```
Comment (gregor) [inline electron/renderer/src/actions/index.js]: Let's keep this in alphabetical order please

File: electron/renderer/src/components/Sidebar.js [hunk 2]
```diff
@@ -66,24 +80,51 @@ const Sidebar = ({
...
+          forceVisible={isContextMenuVisible}
+        />
+      )}
 
     <ContextMenu id="account">
-      <ContextMenuItem onClick={() => window.open('https://wire.com/create-team/?pk_campaign=client&pk_kwd=desktop')}>
+      <ContextMenuItem
+        onClick={() =>
+          window.open(
+            'https://wire.com/create-team/?pk_campaign=client&pk_kwd=desktop'
```
Comment (gregor) [inline electron/renderer/src/components/Sidebar.js]: We need to make this URL environment aware if we have the time to. Code style of these lines also looks off.

File: electron/renderer/src/components/context/ContextMenuTrigger.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import React from 'react';
+
+const ContextMenuTrigger = ({ onClick, forceVisible }) => (
```
Comment (gregor) [inline electron/renderer/src/components/context/ContextMenuTrigger.js]: I would rename this as it is not a generic context menu trigger but the specific one for the add account/create team action.

File: electron/renderer/src/components/Sidebar.js (hunk 2 above)
Comment (AndyTheGiant) [reply electron/renderer/src/components/Sidebar.js]: Let's do this in another PR.

File: electron/renderer/src/components/Sidebar.js [hunk 4]
```diff
@@ -66,24 +80,51 @@ const Sidebar = ({
...
+  </div>
+);
 
-export default Sidebar;
+export default connect(
+  ({ accounts, contextMenuState }) => ({
+    accounts,
+    currentAccentID: (accounts.find(account => account.visible) || {}).accentID,
+    hasCreatedAccount: accounts.some(account => account.userID !== undefined),
+    hasReachedLimitOfAccounts: accounts.length === 3,
```
Comment (gregor) [inline electron/renderer/src/components/Sidebar.js]: Let's be safer in this case and make sure that if someone managed to create more than three account - however that might be possible (changing the content of the database, etc) - the test still works. => `accounts.length >= 3`

**Pull request merged**